core/src/main/proto/spotify_100.proto

```diff
@@ -21,8 +21,8 @@ message Metric {
     // resource "tags" associated to metric.
     map<string, string> resource = 5;
 
-    // Point value
-     Point point =  10;
+    // Distribution type value
+     Value distribution_type_value = 6;
```

Comment:
I think this name a bit confusing - this should be something `value_obj` as it could support either double or distribution.

Reply:
value_obj  :). NP, I will change with the next PR.